packages/schema/lib/schemas/FirehoseWebhookSchema.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const makeSchema = require('../utils/makeSchema');
+
+const FunctionSchema = require('./FunctionSchema');
+
+module.exports = makeSchema(
+  {
+    id: '/FirehoseWebhookSchema',
+    description:
+      'To idenitfy firehose webhooks are their corresponding Zaps. ' +
+      'For Zapier internal webhooks - not publically available.',
```

Comment:
At the moment the answer to that is no. But it is possible this might change, although not any time soon. If we did open it up to devs, we would need to make some changes and add a few more safety checks and docs etc